guides/development-workflow.md

```diff
@@ -190,6 +190,7 @@ To deploy:
   ```
   git fetch
   git checkout master
+  git pull
```

Comment:
@tahb we clearly need to talk about this further - as a whole or partial team. One thing I strongly believe is that any process requiring everyone to have their local tooling configured a certain way is likely to fail.